test/unit/helpers/dom/EnrollTotpBarcodeForm.js

```diff
@@ -68,6 +68,14 @@ define(['./Form'], function (Form) {
 
     clickRefreshLink: function () {
       return this.refreshLink().click();
+    },
+
+    waitForManualSetupLink: function (resolveValue) {
```

Comment:
I can't find where this function is used. If it is not used, please remove it.

Reply:
Removed